infrastructure/sam/MoralPainFunctions/src/main/java/edu/uva/cs/moralpain/typedb/TypeDBPost.java

```diff
@@ -0,0 +1,86 @@
+package edu.uva.cs.moralpain.typedb;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.ArrayList;
+
+import com.vaticle.typedb.client.TypeDB;
+import com.vaticle.typedb.client.api.TypeDBClient;
+import com.vaticle.typedb.client.api.TypeDBSession;
+import com.vaticle.typedb.client.api.TypeDBTransaction;
+import com.vaticle.typedb.client.api.answer.ConceptMap;
+import com.vaticle.typeql.lang.TypeQL;
+import com.vaticle.typeql.lang.query.TypeQLInsert;
+import static com.vaticle.typeql.lang.TypeQL.var;
+
+public class TypeDBPost implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+
+    JSONObject obj = new JSONObject();
+    List<JSONObject> reports = new ArrayList<JSONObject>();
+    JSONParser parser = new JSONParser();
+
+    public APIGatewayProxyResponseEvent handleRequest(final APIGatewayProxyRequestEvent input, final Context context) {
+        Map<String, String> headers = new HashMap<>();
+        headers.put("Content-Type", "application/json");
+        headers.put("X-Custom-Header", "application/json");
+
+        APIGatewayProxyResponseEvent response = new APIGatewayProxyResponseEvent().withHeaders(headers);
+
+        String body = input.getBody();
+
+        try {
+            JSONObject jsonBodyObject = (JSONObject) parser.parse(body);
+            String id = (String) jsonBodyObject.get("id");
+            @SuppressWarnings("unchecked")
+            List<String> selections = (List<String>) jsonBodyObject.get("selections");
```

Comment:
what is this warning?

Reply:
Type safety: Unchecked cast from Object to List<String>